sycl/include/CL/sycl/handler.hpp

```diff
@@ -810,7 +817,13 @@ class __SYCL_EXPORT handler {
         if (Arg[0] >= NumWorkItems[0])
           return;
         Arg.set_allowed_range(NumWorkItems);
-        KernelFunc(Arg);
+        if constexpr (detail::kernelHandlerIsLastElementTypeOfKernel<
```

Comment:
Is it possible to avoid using `if constexpr` outside of __SYCL_DEVICE_ONLY__ region?

Reply:
It is, thanks, fixed.